### DISCUSSION TITLE
Is there any tutorial for using gRPC in goa?

### DISCUSSION THREAD

**Discussion by kuchaguangjie** [goadesign/goa#2770] [General]
The tutorial in the github README page is great, but it's using http.

Is there a similar tutorial for gRPC?

**Comment by raphael** [comment]
The DSL is [well documented](https://pkg.go.dev/goa.design/goa/v3/dsl#GRPC) and that should help. There's also the site https://goa.design and the examples https://github.com/goadesign/examples. Finally there is the slack community (link in the README) where other Goa user may be able to answer specific questions.
